routers/api/packages/generic/generic.go

```diff
@@ -144,7 +145,7 @@ func UploadPackage(ctx *context.Context) {
 
 // DeletePackage deletes the specific generic package.
 func DeletePackage(ctx *context.Context) {
-	err := packages_service.RemovePackageVersionByNameAndVersion(
+	err := common.RemovePackageVersionByNameAndVersion(
```

Comment:
Why the `common` is in "routers" package but not "services"?

Reply:
If you need to resolve the dependency problem, it should be something like `services/packagemanager` to manage all packages.